### ISSUE TITLE
[Q] Usage of the GuiceFXMLLoader

### ISSUE THREAD

**Issue by thatsIch** [cathive/fx-guice#4]
I am currently trying to figure out gow to use your GuiceFXMLLoader. from the JavaDocs, you say, you just need to use

@Inject
private GuiceFXMLLoader loader;

whereever you want, though if I use it outside of my GuiceApplication (which are injected classes) the loader is always null.

I tried to understand it with using one of your smaller (great) examples

App: http://pastebin.com/AekWne6d
View: http://pastebin.com/5a44USap
FXML: http://pastebin.com/1b66QhLp
Error: http://pastebin.com/qMcs2nmM

basically the loader is null, so I get a NPE, so my question is: what to do, to initialize it via @Inject?

Thank you already for reading it :)
